src/OrchardCore/OrchardCore.Abstractions/Json/JOptions.cs

```diff
@@ -31,10 +31,11 @@ public static class JOptions
 
     static JOptions()
     {
+        Base.Converters.Add(new LoginInfoJsonConverter());
+
         Default = new JsonSerializerOptions(Base);
         Default.Converters.Add(new DynamicJsonConverter());
         Default.Converters.Add(new PathStringJsonConverter());
-        Default.Converters.Add(new LoginInfoJsonConverter());
```

Comment:
not work

Reply:
why do you think this does not work?

Reply:
What I mean is that adding the transformer to Default won't work because Yessql's serialization option uses Base instead of Default

Reply:
It should work because we add it in the options which is used by yessql. You can add a test case to prove that it works 

https://github.com/OrchardCMS/OrchardCore/blob/ecd563ce959c2d1b37efe67e8a148333b71c152d/src/OrchardCore/OrchardCore.Data.YesSql/DefaultJsonContentSerializer.cs#L7